### DISCUSSION TITLE
Config.json explanation and options?

### DISCUSSION THREAD

**Discussion by krprice84** [seerge/g-helper#1175] [Q&A]
Hi everyone, I've been searching high and low, and just can't seem to find this anywhere, so excuse me if I've just missed it somehow.

Is there a page or post or anything that can tell me what all of the different options in the config.json file actually do, and what options/variables are valid to use, and what they each do?

i'm struggling to get this app to work exactly how I want it to, but i can't seem to figure out the config file.

Please help!?

**Comment by seerge** [comment]
@krprice84 hello,

All user-editable (and very optional) config params are [explained in readme ](https://github.com/seerge/g-helper#power-user-settings)

The rest are automatically generated by app, just to store it's own settings. You are not supposed to edit them
